machine/wiwynn/andromeda/machine.make

```diff
@@ -0,0 +1,79 @@
+ONIE_ARCH ?= x86_64
+SWITCH_ASIC_VENDOR = BRCM
+
+VENDOR_REV ?= 0
+
+# Translate hardware revision to ONIE hardware revision
+ifeq ($(VENDOR_REV),0)
+  MACHINE_REV = 0
+else
+  $(warning Unknown VENDOR_REV '$(VENDOR_REV)' for MACHINE '$(MACHINE)')
+  $(error Unknown VENDOR_REV)
+endif
+
+# The VENDOR_VERSION string is appended to the overal ONIE version
+# string.  HW vendors can use this to appended their own versioning
+# information to the base ONIE version string.
+## VENDOR_VERSION = .1.1
+
+# Vendor ID -- IANA Private Enterprise Number:
+# http://www.iana.org/assignments/enterprise-numbers
+# Open Compute Project IANA number
+# Wiwynn
+VENDOR_ID = 40092
+
+# Skip the i2ctools and the onie-syseeprom command for this platform
+I2CTOOLS_ENABLE = yes
+
+# The onie-syseeprom command in i2ctools is deprecated.  It is recommended to
+# use the one implemented in busybox instead.  The option intends to provide a
+# quick way to turn off the feature in i2ctools.  The command will be removed
+# from i2ctools in the future once all machines migrate their support of
+# sys_eeprom to busybox.
+#
+# The option is significant when I2CTOOLS_ENABLE is 'yes'
+#
+I2CTOOLS_SYSEEPROM = no
+
+# Set the desired kernel version.
+LINUX_VERSION		= 3.14
+LINUX_MINOR_VERSION	= 27
+
+# Set the desired uClibc version
+UCLIBC_VERSION = 0.9.33.2
```

Comment:
you set this variable twice in this file. here and below on line 73.  only do it once, as it is confusing.